subscription-service/src/main/kotlin/com/egm/stellio/subscription/model/EntityTypeSelector.kt

```diff
@@ -1,10 +1,9 @@
 package com.egm.stellio.subscription.model
 
-import com.egm.stellio.shared.model.ExpandedTerm
 import java.net.URI
 
-data class EntityInfo(
+data class EntityTypeSelector(
```

Comment:
I see no occurrence of `EntityTypeSelector` in the specification

Reply:
I will change that